Libraries/LibWeb/DOM/Position.h

```diff
@@ -62,6 +62,38 @@ class Position {
     unsigned m_offset { 0 };
 };
 
+class Range {
```

Comment:
Let's make DOM::Range follow the specced Range type as closely as possible: https://developer.mozilla.org/en-US/docs/Web/API/Range :)

(Also note that it needs to be reference-counted since it will be exposed to JavaScript.)